src/tribler-common/tribler_common/tests/test_patch_import.py

```diff
@@ -26,13 +25,13 @@ async def test_mock_import_import_real_lib():
 @patch_import(['time'])
 async def test_mock_import_not_strict():
     import time
-    assert not isinstance(time, MagicMock)
+    assert not isinstance(time, Mock)
```

Comment:
**Issue**: after reverting back `patch_import` to the using of `MagicMock` class, this line also should be reverted.